static/app/views/issueDetails/traceTimeline/useTraceTimelineEvents.tsx

```diff
@@ -164,7 +160,9 @@ export function useTraceTimelineEvents({event}: UseTraceTimelineEventsOptions):
         'project.name': '',
         timestamp: event.dateCreated!,
         title: event.title,
+        // The event variable comes from a call to the issues endpoint
         transaction: '',
+        // XXX: Typing is not working here; this should event.type
```

Comment:
This is a very similar situation as to what I noticed in #74010

I will not be fixing it in this PR.